paddle/fluid/operators/controlflow/recurrent_op_helper.cc

```diff
@@ -72,8 +72,10 @@ static void FindAllOpAndGradOp(const framework::ProgramDesc &program,
   OpVariantSet &ops = op_and_grad_op->first;
   OpVariantSet &grad_ops = op_and_grad_op->second;
 
-  PADDLE_ENFORCE_GE(ops.size(), grad_ops.size(),
-                    "There are extra grad ops in the graph or program");
+  PADDLE_ENFORCE_GE(
+      ops.size(), grad_ops.size(),
+      platform::errors::InvalidArgument(
+          "There are more grad ops than forward ops in the graph or program"));
```

Comment:
It could be better to add `.` at last of sentence.

Reply:
May be better to  print `op.size()` and `grad_ops.size()` like
```
There are more grad ops than forward ops in the graph or program, 
the number of ops is %d and the number of grad_ops is %d ...
```

Reply:
Done